src/test/scala/uk/gov/hmrc/perftests/itsass/RequestsHelper.scala

```diff
@@ -36,18 +42,22 @@ object RequestsHelper extends ServicesConfiguration {
   val dividendsUrl: String = personalIncomeBaseUrl + s"/income-through-software/return/personal-income/$taxYear/dividends"
   val interestUrl: String = personalIncomeBaseUrl + s"/income-through-software/return/personal-income/$taxYear/interest"
   val giftAidUrl: String = personalIncomeBaseUrl + s"/income-through-software/return/personal-income/$taxYear/charity"
-  val employmentsUrl: String = employmentFrontendBaseUrl + s"/income-through-software/return/employment-income/$taxYear"
+  val employmentsUrl: String => String =(taxYear:String) => employmentFrontendBaseUrl + s"/income-through-software/return/employment-income/$taxYear"
 
   val csrfPattern: String = """<input type="hidden" name="csrfToken" value="([^"]+)"/>"""
   val untaxedAccountPattern: String = s"""/income-through-software/return/personal-income/$taxYear/interest/add-untaxed-uk-interest-account/([^"]+)"""
   val taxedAccountPattern: String = s"""/income-through-software/return/personal-income/$taxYear/interest/add-taxed-uk-interest-account/([^"]+)"""
+  val employmentIdPattern: String = s"""/income-through-software/return/employment-income/$taxYearEOY/employer-name([^"]+)"""
```

Comment:
does this mean its saving the / aswell as the employmentId? is it worth removing that so you only have the employment id value?

Reply:
updated